README.md

```diff
@@ -1,24 +1,26 @@
 [![Licensed under the MIT License](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/microsoft/spatialaudio-unity/blob/master/LICENSE)
 
 # Introduction 
-This repository provides plugins and tools for integrating spatial audio and acoustics into your Unity 3D applications and games. This includes:
-- A HoloLens 2 spatializer plugin which uses [Windows Sonic API](https://docs.microsoft.com/en-us/windows/win32/coreaudio/spatial-sound) to enable hardware offload of spatial audio processing, freeing up the CPU for your application.
-- A sample Unity application that demonstrates proper usage of the HoloLens 2 spatializer plugin.
-- Source code for the [Project Acoustics](http://aka.ms/acoustics) spatializer plugin.
+This repository provides plugins and tools for integrating spatial audio and acoustics into your Unity 3D applications and games  
+- A **cross-platform spatializer plugin (supported on Windows and Android)** that uses FLEX DSP algorithms for highly efficient spatial audio processing.

```

Comment:
I don't think that we should mention FLEX publicly just yet.  It's really just an internal codename and I'd prefer not to disclose as of yet.

I think that it is sufficient to say "that uses highly efficient spatial audio DSP processing algorithms."